deploy/internal/pvc-agent.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  labels:
+    app: noobaa
+  name: noobaa-pv-claim
+spec:
+  storageClassName: standard
```

Comment:
I'm not sure that `standard` is a predifined storage class. If you just ommit the storageclass I think the default stoarge class (defined in the cluster) will be used

https://kubernetes.io/docs/concepts/storage/storage-classes/#the-storageclass-resource